.github/workflows/docker-publish.yml

```diff
@@ -0,0 +1,77 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Publish Image to ghcr.io
+
+# This workflow uses actions that are not certified by GitHub.
+# They are provided by a third-party and are governed by
+# separate terms of service, privacy policy, and support
+# documentation.
+
+on:
+  schedule:
+    - cron: '10 16 * * *'
+  push:
+    branches: [ main ]
+    # Publish semver tags as releases.
+    tags: [ 'v*.*.*' ]
+  pull_request:
+    branches: [ main ]
+
+env:
+  # Use docker.io for Docker Hub if empty
+  REGISTRY: ghcr.io
+  # github.repository as <account>/<repo>
+  AUTHZ_SERVER: ${{github.repository_owner}}/authz-server
```

Comment:
Should we name this registry-authz-server ?

Reply:
Do you think that's better? I'm not sure. The authz-server is completely generic - it has nothing specific to the registry server in it, but I'd be ok with calling this "registry-authz-server" if you think we should prefix all of our containers with "registry-"

Reply:
When this builds it would build on the container registry as gchr.io/apigee/authz-server  which may give an impression that this is something that can be used with any Apigee component.

Reply:
OK, based on this and our conversation I've changed the container name to `registry-authz-server`.